### PR TITLE
ODIM 3556 add connection method configuration values

### PR DESCRIPTION
**Pull request by irappa-pattar**

### CODE REVIEW

**Review by jeevan-kamkar — APPROVED** [review]

File: lib-utilities/config/config_test.go [hunk 1]
```diff
@@ -131,7 +131,25 @@ func TestSetConfiguration(t *testing.T) {
...
+				"ConnectionMethodType":"Redfish",
+				"ConnectionMethodVariant":"Compute:GRF_v1.0.0"
+			},
+			{
+				"ConnectionMethodType":"Redfish",
+				"ConnectionMethodVariant":"Fabric:CFM_v1.0.0"
+			},
+			{
+				"ConnectionMethodType":"Redfish",
+				"ConnectionMethodVariant":"Storage:STG_v1.0.0"
```
Comment (Bharath-KKB) [inline lib-utilities/config/config_test.go:142]: Storage is missing in SupportPluginTypes.

File: lib-utilities/config/config_test.go [hunk 2]
```diff
@@ -131,7 +131,25 @@ func TestSetConfiguration(t *testing.T) {
...
+				"ConnectionMethodType":"Redfish",
+				"ConnectionMethodVariant":"Compute:iLO_v1.0.0"
+			},
+			{
+				"ConnectionMethodType":"Redfish",
+				"ConnectionMethodVariant":"Compute:GRF_v1.0.0"
+			},
+			{
+				"ConnectionMethodType":"Redfish",
+				"ConnectionMethodVariant":"Fabric:CFM_v1.0.0"
```
Comment (Bharath-KKB) [inline lib-utilities/config/config_test.go]: Dont add this.

File: lib-utilities/config/config_test.go [hunk 3]
```diff
@@ -131,7 +131,25 @@ func TestSetConfiguration(t *testing.T) {
                 "Fabrics",
                 "Managers"
 		],
-		"SupportedPluginTypes" : ["Compute", "Fabric"]
+		"SupportedPluginTypes" : ["Compute", "Fabric"],
+		"ConnectionMethodConf":[
+			{
+				"ConnectionMethodType":"Redfish",
```
Comment (Bharath-KKB) [inline lib-utilities/config/config_test.go]: Dont add this

**Review by Bharath-KKB — CHANGES_REQUESTED** [review]
address comments

File: lib-utilities/config/config_test.go (hunk 3 above)
Comment (irappa-pattar) [reply lib-utilities/config/config_test.go]: removed

File: lib-utilities/config/config_test.go (hunk 2 above)
Comment (irappa-pattar) [reply lib-utilities/config/config_test.go]: removed

File: lib-utilities/config/config_test.go (hunk 1 above)
Comment (irappa-pattar) [reply lib-utilities/config/config_test.go:142]: added

**Review by jeevan-kamkar — APPROVED** [review]
LGTM

**Review by a-ajith — APPROVED** [review]

**Review by Bharath-KKB — APPROVED** [review]

**Pull request merged**